pkg/cmd/root.go

```diff
@@ -57,7 +57,7 @@ func Execute() (*types.CmdValues, error) {
 
 func addProviderFlags(cmd *cobra.Command, vals *types.CmdValues) *cobra.Command {
 
-	cmd.Flags().StringSliceVarP(&vals.Files, "files", "f", []string{"docker-compose.yml"}, "Provide docker-compose files, comma separated.")
+	cmd.Flags().StringSliceVarP(&vals.Files, "files", "f", nil, "Provide docker-compose files, comma separated.")
```

Comment:
putting `docker-compose.yml` here will be helpful to user, because it will show up in automatic help.